### DISCUSSION TITLE
"Used by counter" missing after renaming project

### DISCUSSION THREAD

**Discussion by janpio** [community/community#21763] [New to GitHub]
<p>Hey,</p>
<p>we just renamed our main repository from <a href="https://github.com/prisma/prisma" rel="nofollow noopener">https://github.com/prisma/prisma2</a> to <a href="https://github.com/prisma/prisma" rel="nofollow noopener">https://github.com/prisma/prisma</a>. Unfortunately somehow during move we lost our “Used by counter” dropdown in the settings and now can not change the package from <code>prisma</code> to <code>@prisma/cli</code> (which is the new package we updated to last week).</p>
<p>Missing something, does this need some time or des this have to be updated manually?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/janpio">@janpio</a>,</p>
<p>Thanks for being here! I’ve triggered it on our end it might take a few moments to update. </p>

**Comment by janpio** [comment]
<p>Unfortunately the dropdown is still invisible on the repo settings page. Am I looking in the wrong place? Or can you look at it again please?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/janpio">@janpio</a>,</p>
<p>Thanks for the ping! Are you logged in, when you are looking at the repo?</p>

**Comment by janpio** [comment]
<p>I am, without I could not see the settings.</p>
<p>But the dropdown is now visible and I could change the setting. Thanks!</p>

**Comment by janpio** [comment]
<p>Pinging you here directly <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> instead of opening a new topic (let me know if you don’t like that):</p>
<p>Can you click the same button for <a href="https://github.com/prisma/prisma-client-js" rel="nofollow noopener">https://github.com/prisma/prisma-client-js</a>?</p>
<p>Hoping to lis <code>@prisma/client</code> there: <a href="https://github.com/prisma/prisma-client-js/blob/1ba724ccee15535489257984265798f70c52e7a5/packages/photon/package.json#L2" rel="nofollow noopener">https://github.com/prisma/prisma-client-js/blob/1ba724ccee15535489257984265798f70c52e7a5/packages/photon/package.json#L2 </a></p>
<p>Thanks.</p>

**Comment by AndreaGriffiths11** [comment]
<p>No worries <a class="mention" href="/janpio">@janpio</a>, here is fine and done.</p>
